.github/workflows/docker.yml

```diff
@@ -96,12 +96,22 @@ jobs:
       - name: Run test suite
         run: docker-compose run sealog-server npm test
 
-      - name: Run server
+      - name: Run server and Postman tests
         run: |
           docker-compose up -d
           sleep 10
           curl http://localhost:8000/sealog-server > /tmp/banner.txt
           grep "Welcome to sealog-server\!" /tmp/banner.txt
+
+          # Install Newman for running Postman tests
+          npm install -g newman
+
+          # Run Postman tests
+          newman run "integration-tests/api_tests.postman_collection.json" \
+            --env-var "baseURL=http://localhost:8000" \
+            --env-var "user=${{ secrets.API_TEST_USER }}" \
+            --env-var "password=${{ secrets.API_TEST_PASSWORD }}"
```

Comment:
Do we need to use secrets here, since this is an ephemeral instance?

Reply:
Yeah probably not, force of habit